cookbooks/bcpc/recipes/mysql.rb

```diff
@@ -100,13 +100,29 @@
           '-o Dpkg::Options::="--force-confold"'
 end
 
+# This package is needed to be a donor in the SST process.
+package 'percona-xtrabackup' do
+  action :install
+end
+
 directory '/var/run/mysql' do
   owner 'mysql'
   mode 0775
 end
 
+#
+# On Ubuntu, supports[:status] defaults to false, which causes chef to
+# use 'pgrep -f' to look for the running service.  This will always
+# succeed because java processes include MySQL in their classpaths.
+#
+# Setting it to true induces chef to check service status through the
+# 'service' command.
+#
 service 'mysql' do
   action [:enable, :start]
+  supports({restart: false,
+            reload: false,
```

Comment:
Up above in this recipe the service is notified with `:reload` often. Is this supported by the service or should those be changed to `:restart`?